addons/hashes/fnc_hashFilter.sqf

```diff
@@ -0,0 +1,59 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_hashFilter
+
+Description:
+    Iterate through all keys and values in a Hash.
+    If function returns true, the key is removed from the hash.
```

Comment:
I don't know - it seems fine as it is (remove if true)

- `count` will count an array element if the expression is `true`
- `select` will use an array element if the expression is `true`
- imagine two CBA hash functions `hashCount` and `hashSelect` that would do for the same  for hashes

--> `hashFilter` modifying the hash when the function provided returns `true` sounds consistent